homeassistant/components/samsungtv/media_player.py

```diff
@@ -35,6 +34,7 @@
 from homeassistant.util import dt as dt_util
 
 from .const import CONF_MANUFACTURER, CONF_MODEL, CONF_ON_ACTION, DOMAIN, LOGGER
+from .remote import SamsungTVRemote
```

Comment:
Platforms are not allowed to depend on each other.

Reply:
It would be ok to have a non entity remote that can be used in both platforms.

Reply:
Sure, that would work.

Out of interest, is the mentioned "not allowed" rule documented somewhere, with reasoning elaborated?

Reply:
I don't know.